docs/api/effector/Effect.md

```diff
@@ -211,3 +211,36 @@ const isLoading = createStore(false)
   .on(fetchApi.done, () => false)
   .on(fetchApi.fail, () => false)
 ```
+
+### `.finally`
+
+_Event_ triggered when promise from _thunk_ is *resolved*, *rejected* or thunk throws.
```

Comment:
Could you change all references to `thunk` to `handler`? Handler is a term from effect's constructor `createEffect({handler})`